test/tests/functional/pbs_alps_release_tunables.py

```diff
@@ -151,10 +149,11 @@ def test_alps_release_jitter(self):
                 retry -= 1
                 # variable 'out' is a list of tuples and every second element
                 # in a tuple is the matched log message
-                time_prev = get_epoch(out[0][1])
+                time_prev = self.get_epoch(out[0][1])
                 for data in out[1:]:
-                    time_current = get_epoch(data[1])
-                    self.assertLessEqual(time_current - time_prev, max_delay,
+                    time_current = self.get_epoch(data[1])
+                    self.assertLessEqual(int(time_current - time_prev),
```

Comment:
I'm still not understanding why you are doing an int() here.  I must be missing something.  Can you please explain?  Thanks.